oluwajuwon/Fast-Food-Fast
# Feedback ui template

### What does this pull request do?
This pull request creates a cart and checkout feature to add multiple food items

### Description of the Task to be completed?
Users should be able to add multiple items to a single order

### How should this be manually tested?
Users can test it by:
 - Visit https://oluwajuwon.github.io/Fast-Food-Fast/UI/index.html
 - When you click on an item to add it to cart
 - It should add the item to the cart of selected items
 - When the user checks out
   - If the user is not logged in, it should take the user to the login page
   - If the user is logged in, it should take the user to the checkout page


### What are the relevant pivotal tracker stories?
The relevant pivotal story is:

Implement a cart feature for multiple items  - https://www.pivotaltracker.com/story/show/160948567 

### Screenshots
![10](https://user-images.githubusercontent.com/29441898/46414927-1f1ff580-c71c-11e8-895a-1b5408f95369.PNG)

![11](https://user-images.githubusercontent.com/29441898/46415417-2b588280-c71d-11e8-9426-360f506e68d6.PNG)
